README.md

```diff
@@ -10,8 +10,64 @@ The website uses the LAA Legal Adviser API. The code is hosted at https://github
 
 The REST API is https://prod.laalaa.dsd.io/.
 
+## Dependencies
+- [docker](https://www.docker.com/)
+
+## Installation for development
+
+### Tips for developing with docker containers
+
+Code editing - You can edit the code on your local disk, with a local editor, as normal. (You don't have to edit the 
+files inside the Docker container, because your local directory is mounted into container.) When you save a file, it 
+becomes present in the container immediately, and the server restarts.
+
+Browsing the app - Point your local browser at http://localhost:8013/
+
+Log output - watch the output generated by the running app using: `docker logs fala -f` or `docker attach fala`
+
+Please ensure your internet network such as your VPN does not interfere with your build.
+
+To run container:
+
+`./run_local.sh`
```

Comment:
These 3 lines are the key instruction of how to run Fala in Docker. I should move it up, out of the 'tips', so it's the main instruction of the 'Installation for development' section

Reply:
Moved up out of tips into `Docker` header sections.